### ISSUE TITLE
Volumes is not a field of struct type *docker.startContainerTemplate

### ISSUE THREAD

**Issue by joelmoss** [mitchellh/packer#1512]
Trying to build is giving me this error. Any ideas please?

```
==> docker: Starting docker container...
2014/09/19 16:42:24 ui: ==> docker: Starting docker container...
==> docker: Error running container: template: tpl11:1:2: executing "tpl11" at <.Volumes>: Volumes is not a field of struct type *docker.startContainerTemplate
2014/09/19 16:42:24 ui error: ==> docker: Error running container: template: tpl11:1:2: executing "tpl11" at <.Volumes>: Volumes is not a field of struct type *docker.startContainerTemplate
Build 'docker' errored: Error running container: template: tpl11:1:2: executing "tpl11" at <.Volumes>: Volumes is not a field of struct type *docker.startContainerTemplate
2014/09/19 16:42:24 ui error: Build 'docker' errored: Error running container: template: tpl11:1:2: executing "tpl11" at <.Volumes>: Volumes is not a field of struct type *docker.startContainerTemplate
2014/09/19 16:42:24 packer-command-build: 2014/09/19 16:42:24 Builds completed. Waiting on interrupt barrier...
2014/09/19 16:42:24 machine readable: error-count []string{"1"}

==> Some builds didn't complete successfully and had errors:
2014/09/19 16:42:24 ui error: 
==> Some builds didn't complete successfully and had errors:
2014/09/19 16:42:24 machine readable: docker,error []string{"Error running container: template: tpl11:1:2: executing \"tpl11\" at <.Volumes>: Volumes is not a field of struct type *docker.startContainerTemplate"}
--> docker: Error running container: template: tpl11:1:2: executing "tpl11" at <.Volumes>: Volumes is not a field of struct type *docker.startContainerTemplate
2014/09/19 16:42:24 ui error: --> docker: Error running container: template: tpl11:1:2: executing "tpl11" at <.Volumes>: Volumes is not a field of struct type *docker.startContainerTemplate
```

**Comment by mitchellh** [comment]
The `{{.Volumes}}` parameter is no longer needed. Simply remove it. Sorry for the BC change, I'll be more careful in the future!
